src/slices.py

```diff
@@ -12,26 +12,27 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+t = a[slice(1, 2)]
```

Comment:
Though using `slice` is basically the same thing, this file was intending for you to use slice notation. Check out this link from the comment at the top of the file https://stackoverflow.com/a/509295